spec/integration/integration_helper.rb

```diff
@@ -5,6 +5,7 @@
 begin
   require 'active_support'
 rescue LoadError
+  print 'No "active_support" loaded'
```

Comment:
This was here to run the integration tests for both datamapper as well as activerecord/activesupport. If support for datamapper has been dropped, the begin/rescue block as well as the `defined?` check cah be removed.